lib/grover.rb

```diff
@@ -23,7 +23,7 @@ class Processor < Schmooze::Base
     dependencies puppeteer: 'puppeteer'
 
     def self.launch_params
-      ENV['GROVER_NO_SANDBOX'] == 'true' ? "{args: ['--no-sandbox', '--disable-setuid-sandbox']}" : '{}'
+      ENV['GROVER_NO_SANDBOX'] == 'true' ? "{args: ['--no-sandbox', '--disable-setuid-sandbox']}" : '{args: []}'
```

Comment:
not sure this change is necessary if you instead just assign the args on L46 ?? Why have a default and concat if they're always empty?